covasim/webapp/package.json

```diff
@@ -4,15 +4,30 @@
   "private": true,
   "description": "Agent-based model",
   "main": "index.js",
-  "scripts": {},
+  "scripts": {
+    "build": "webpack --config webpack.config.js --mode=development",
+    "build:prod": "webpack --config webpack.config.js --mode=production"
+  },
   "author": "Institute for Disease Modeling",
   "license": "CC-BY-SA-4.0",
   "dependencies": {
-    "bootstrap": "4.4.1",
+    "bootstrap": "^4.4.1",
     "bootstrap-vue": "2.11.0",
     "file-saver": "2.0.2",
-    "plotly.js-dist": "1.53.0",
```

Comment:
Please add this back. It's needed for dependency tracking.

Reply:
Updated to use plotly.js-cartesian-dist-min for smaller bundle size

Reply:
Updated to use plotly.js-cartesian-dist-min for smaller bundle size